packages/@adobe/spectrum-css-temp/components/stepper/index.css

```diff
@@ -65,10 +65,6 @@ governing permissions and limitations under the License.
 
   padding: var(--spectrum-button-padding-y) calc(var(--spectrum-button-primary-padding-x) - var(--spectrum-button-primary-border-size));
 
-  /* @hack: fix button text vertical alignment at 14px */
-  padding-block-end: calc(var(--spectrum-button-padding-y) + 1px);
-  padding-block-start: calc(var(--spectrum-button-padding-y) - 1px);
```

Comment:
@snowystinger do you remember what exactly this references? Found the following commit: https://github.com/adobe/react-spectrum/commit/398a051711d90a1231393cee5ffd768488fa1e06, not sure what pointer issue it is referencing

Reply:
o, it was this issue https://github.com/adobe/react-spectrum/pull/1523/files#diff-baa7d39d9b1465a7d8ce5b1760066982303b4f8a93fc797af787879bef41bfb4R94

Reply:
this hack is probably from spectrum-css button code?

Reply:
gotcha, well it seems to work for me in safari still so hopefully we are fine here

Reply:
yep, it's a div, you won't have a problem functionally